eo-parser/src/main/java/org/eolang/parser/ParsingTrain.java

```diff
@@ -110,34 +121,41 @@ public ParsingTrain() {
      */
     ParsingTrain(final String... sheets) {
         super(
-            new TrLambda(
-                new TrFast(
-                    new TrLambda(
-                        new TrLogged(
-                            new TrClasspath<>(sheets).back(),
-                            ParsingTrain.class,
-                            Level.FINEST
+            new TrJoined<>(
```

Comment:
@maxonfjvipon Why do we need to add `move-versions-deeper.xsl` to the pipeline as a separate `TrDefailt`? Can we just add `move-versions-deeper.xsl` to the `ParsingTrain.SHEETS`?

Reply:
@volodya-lombrozo I explained this in the description of transformation.
1. One application of transformation moves version one level deeper
2. Because of point 2 It should be applied many times until xml after applying is not changed anymore (until we get the deep)

That's why it stays at the end and is wrapped in `TrEndless`

Reply:
@maxonfjvipon Oh, I see. It's hard.